source/edge-server/configure.txt

```diff
@@ -433,6 +307,44 @@ server, as well as errors that may have occurred.
          }
```

Comment:
Can't comment on it, but I don't think we need the example status response object in an example admonition